modules/smart-agent_kubernetes-velero/detectors-velero.tf

```diff
@@ -21,6 +21,8 @@ EOF
     parameterized_subject = var.message_subject == "" ? local.rule_subject : var.message_subject
     parameterized_body    = var.message_body == "" ? local.rule_body : var.message_body
   }
+
+  max_delay = var.velero_scheduled_backup_missing_max_delay
```

Comment:
Not directly related to this PR, is it intentional to have no heartbeat detector for Velero metrics? Are metrics too sparse?

Reply:
I can answer for this one. Indeed, there is no "regular" metric reporting so it is basically not possible, as for lambda for example